src/Dapr.Actors/Runtime/ActorReminder.cs

```diff
@@ -124,6 +124,34 @@ internal ActorReminder(ActorReminderOptions options)
             this.Period = options.Period;
             this.Ttl = options.Ttl;
         }
+        
+        /// <summary>
+        /// Initializes a new instance of <see cref="ActorReminder" />.
+        /// </summary>
+        /// <param name="actorType">The actor type.</param>
+        /// <param name="actorId">The actor id.</param>
+        /// <param name="name">The reminder name.</param>
+        /// <param name="state">The state associated with the reminder.</param>
+        /// <param name="dueTime">The reminder due time.</param>
+        /// <param name="period">The reminder period.</param>
+        /// <param name="repetitions">The number of times reminder should be invoked.</param>
+        public ActorReminder(
+            string actorType,
+            ActorId actorId,
+            string name,
+            byte[] state,
+            TimeSpan dueTime,
+            TimeSpan period,
+            int repetitions)
+            : this(actorType, actorId, name, state, dueTime, period)
+        {
+            if (repetitions == 0 || repetitions < -1)
```

Comment:
This statement feels odd to me. -1 is a special case I'm guessing? Would we be calling it with that at any time instead of just not setting it? Couldn't we just say anything <0 is a valid way to say infinite repetitions?

Reply:
Fixed